workflow/config_resources/config.base

```diff
@@ -18,6 +18,15 @@ export DO_FCST=${DO_FCST:-true}
 export DO_POST=${DO_POST:-true}
 export DO_IC_LBC=${DO_IC_LBC:-true}
 
+#
+if ${DO_CLEAN:-false} && ! ${REALTIME:-false}; then
+  # when DO_CLEAN is set to true in retros, if STMP_CLEAN_HRS, COM_CLEAN_HRS and LOG_CLEAN_HRS
+  # are not specified, use the following values
+  export STMP_CLEAN_HRS=${STMP_CLEAN_HRS:-24}
+  export COM_CLEAN_HRS=${COM_CLEAN_HRS:-2400}  # 100 days
+  export LOG_CLEAN_HRS=${LOG_CLEAN_HRS:-2400}
+fi
+
```

Comment:
@guoqing-noaa, what is this time relative to? For example, 

1. Is it checking `mtime` in which case it would only keep data that is 24 hours old? 
2. Or is it checking relative to the cycle time that is currently running so that when in retro mode I'll have to cycle through a 24 hour period before it starts deleting? 
3. Or is it comparing against the actual cycle date so that when in retro mode all the cleanup jobs automatically run because we are already 1 year past the retro period?

Reply:
@delippi Good questions.
The clean task runs each cycle, and the `*_CLEAR_HRS` is w.r.t the cycle time at the running cycle.
If you want to purge stmp more aggressively, you can set `STMP_CLEAN_HRS` to for example, 3 or even 1